components/organisms/Layout.js

```diff
@@ -26,8 +26,15 @@ export const Layout = ({
         </div>
 
         <div className="layout-container">
-          <a href="#">Français</a>
-          <SearchBar placeholder={"Search Canada.ca"} dataCy={"search-bar"} />
+          <Link key={language} href={asPath} locale={language}>
```

Comment:
Same thing as the comment above, we should make this such that I can pass in the alternate path from the pages.

For example the french language page for the home page might be `/fr` but lets say for a page `/our-first-blog-post` it might just be `/notre-premier-blog`

Reply:
Layout now has a prop `langUrl` through which an alternate URL can be passed for pages using the Layout component.